mesa/suite/trixie/debian/patches/0001-PVR-changes.patch

```diff
@@ -1,43 +0,0 @@
-From: Andrew Davis <afd@ti.com>
-Date: Sun, 3 Mar 2024 11:32:16 +0000
-Subject: [PATCH] PVR Changes
-
----
- src/gallium/targets/dri/meson.build | 7 ++++++-
- src/gallium/targets/dri/target.c    | 3 +++
- 2 files changed, 9 insertions(+), 1 deletion(-)
-
-diff --git a/src/gallium/targets/dri/meson.build b/src/gallium/targets/dri/meson.build
-index be134700698..4d6e1059f3b 100644
---- a/src/gallium/targets/dri/meson.build
-+++ b/src/gallium/targets/dri/meson.build
-@@ -117,7 +117,12 @@ foreach d : [[with_gallium_kmsro, [
-              [with_gallium_zink, 'zink_dri.so'],
-              [with_gallium_d3d12, 'd3d12_dri.so'],
-              [with_gallium_asahi, 'asahi_dri.so'],
--             [with_gallium_pvr, 'pvr_dri.so'],
-+             [with_gallium_pvr, [
-+               'pvr_dri.so',
-+               'tilcdc_dri.so',
-+               'omapdrm_dri.so',
-+               'tidss_dri.so',
```

Comment:
Without this patch, how is `tidss_dri.so` generated? We do that in the Yocto with the following:

```
PVR_DISPLAY_CONTROLLER_ALIAS ??= "tidss"
PACKAGECONFIG[pvr] = "-Dgallium-pvr-alias=${PVR_DISPLAY_CONTROLLER_ALIAS},"
```
I see no equivalent here. You should add `-Dgallium-pvr-alias=tidss` to the build string.